torch/jit/unsupported_tensor_ops.py

```diff
@@ -0,0 +1,58 @@
+import torch.jit
+import inspect
+import textwrap
+from textwrap import dedent
+from torch._six import PY2
+
+def execWrapper(code, glob, loc):
+    if PY2:
+        exec(code) in glob, loc
+    else:
+        exec(code, glob, loc)
+
+def _gen_unsupported_methods_properties():
+    tensor_attrs = set(filter(lambda x: x[0] != "_", dir(torch.Tensor)))
+    tensor = torch.tensor([2])
+    funcs_template = dedent('''
+    def func(x):
+        return x.{op}()
+    ''')
+
+    deprecated_apis = set(["volatile", "resize", "reinforce", "new", "name", "map2_", "has_names", "grad_fn", "resize_as"])
```

Comment:
Also where is this list coming from?

Reply:
These are the undocumented apis & properties. I could try to expose the list here: https://github.com/pytorch/pytorch/blob/master/test/test_docs_coverage.py#L83